src/store.js

```diff
@@ -59,24 +68,34 @@ const store = new Vuex.Store({
     },
     setSortedColumn(state, column) {
       state.sortedColumn = column;
+    },
+    setCurrentPath(state, path) {
+      state.currentPath = path;
+    },
+    resetAttributesPage(state) {
```

Comment:
some components share some state implied in fetching the data, then it is compulsory reset the data when a component is destroyed